### DISCUSSION TITLE
I am a student of the Azerbaijan Technical University and this university is in your program, but I cannot access your Student developer pack.

### DISCUSSION THREAD

**Discussion by RzayevTamerlan** [community/community#68925] [GitHub Education]
### Hi everyone,

Good day everyone I hope you are doing well. My name is Tamerlan and I am a student and am studying at the Azerbaijan Technical University in the Faculty of Computer Engineering in my second year. I interested in and studying in the direction of Full-stack development. I recently learned about your student program and was eager to join you. But when I try to submit an application, your service requires my email with the end - student.aztu.edu.az. I asked the tutor and the dean's office if they give us mail with this ending and they told me no. Because of this, I cannot join your student program. Please consider this situation and remove the email requirement when selecting Azerbaijan Technical University. Thank you for your attention.
![Screenshot 2023-09-28 112423](https://github.com/community/community/assets/129269486/a2dc97a9-41bd-4160-9b5a-07dfa17d9f98)

**Comment by samus-aran** [comment]
Hi @RzayevTamerlan 👋 

As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient. In this case it would be the Academic email(s) which has been already successfully associated with this school.
We would recommend to you speak to your institution again regarding to acquiring an email address to proceed.

**Reply by RzayevTamerlan** [reply]
Good afternoon again. I asked the tutor and the dean about issuing mail and they both said that this university does not provide such an opportunity to its students. Can I send you a selfie with my student card as proof that I study at this university?

**Reply by samus-aran** [reply]
Hi @RzayevTamerlan 

All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants. 

We have evidence that this school does provide academic emails which has been verified. If you believe that these school details are in correct then (in your open support ticket) you should provide evidence to support if your school has additional academic domains, has changed its academic email issuing policy, or does not provide academic email such as:

- A link to a publicly-accessible page on the school website which lists their academic domain(s)
- An affidavit from your school on school letterhead detailing their current academic email issuing policy

Please note that screenshots of emails purportedly sent by the school to non-academic webmail addresses are not considered to be reliable supporting documentation.

Hope this helps 👍

**Comment by RzayevTamerlan** [comment]
During this process, it was discovered that the university actually provides student email.
